source/bdDetect.py

```diff
@@ -605,7 +612,7 @@ def _getDriverDict(self) -> DriverDictT:
 			ret = _driverDevices[self._driver] = DriverDictT(set)
 			return ret
 
-	def addUsbDevices(self, type: DeviceType, ids: Set[str]):
+	def addUsbDevices(self, type: DeviceType, ids: Set[str], useAsFallBack: bool = False):
 		"""Associate USB devices with the driver on this instance.
 		:param type: The type of the driver.
 		:param ids: A set of USB IDs in the form C{"VID_xxxx&PID_XXXX"}.

```

Comment:
Please at a description of the new `useAsFallBack` parameter to the doc string

Reply:
Thanks! I added description of useAsFallBack